common/core/overlaylayer.cpp

```diff
@@ -61,18 +61,22 @@ int32_t OverlayLayer::ReleaseAcquireFence() const {
 OverlayBuffer* OverlayLayer::GetBuffer() const {
   if (imported_buffer_->buffer_.get() == NULL)
     ETRACE("hwc layer get NullBuffer");
+
   return imported_buffer_->buffer_.get();
 }
 
+std::shared_ptr<OverlayBuffer>& OverlayLayer::GetSharedBuffer() const {
+  return imported_buffer_->buffer_;
+}
+
 void OverlayLayer::SetBuffer(HWCNativeHandle handle, int32_t acquire_fence,
                              ResourceManager* resource_manager,
                              bool register_buffer, HwcLayer* layer) {
   std::shared_ptr<OverlayBuffer> buffer(NULL);
 
-  // FIXME: Enable cache again.
-  /*if (resource_manager && register_buffer) {
+  if (resource_manager && register_buffer) {
```

Comment:
Wait, this looks like more than a warning fix. If you are enabling some feature then that should be made clear in the commit message.

Reply:
That's not part of the patch?
https://github.com/intel/IA-Hardware-Composer/pull/272/files
I think you're looking at a pre git pull rebase patch

Reply:
hmm for some reson pre git pull rebase patch is here....